spec/models/authentication_spec.rb

```diff
@@ -6,13 +6,15 @@
 
   context "when an authentication exists" do
     it "is returned by by_provider_and_uid" do
-      value = Authentication.by_provider_and_uid(authentication.provider, authentication.uid)
+      value = Authentication.find_by_provider_and_uid(
```

Comment:
Those are deprecated. In Rails 4 you'd use `find_by(provider: blah, uid: blah)`.

Still don't need to redefine them though.